src/httprx.js

```diff
@@ -26,7 +28,7 @@ const request = httpClient => (options, data, isBinary) => new rx.Observable(obs
         } else {
             data.pipe(req)
                 .on('error', error => {
-                    console.log(error);
+                    throw new Error(`Request failed: ${er.message}`);
```

Comment:
Same as in the two other changes, I feel that we should use the stream.error. Maybe we should use exceptions as you suggest.